### DISCUSSION TITLE
Web Socket/MCDU Code?

### DISCUSSION THREAD

**Discussion by Rosmanczyk** [flybywiresim/aircraft#7292] [Help]
Where exactly can the MCDU code be found? More specifically: the generated text lines or variables and events?
Would it be possible to create a small app that draws each line as in the original MCDU but creates the freedom of arranging the text as needed for any type of screen?

If anyone knows someone with this skillset please let me know. Thank you a lot!

**Comment by tracernz** [comment]
The complete code for the client side of the remote MCDU is https://github.com/flybywiresim/a32nx/tree/master/src/mcdu-server/client

**Reply by Rosmanczyk** [reply]
Thanks a lot for the link! Do you by accident know someone that is familiar with this code and does work for hire?
